generate-keys.sh

```diff
@@ -21,8 +21,10 @@ if [ -d keys ] ; then
 fi
 mkdir keys
 
+
+KEYNAME="$(grep "keyname:" ./aws.yaml | awk '{ print $2 }')"
```

Comment:
This is an effective way of parsing the yaml in this case, but completely ignores the yaml syntax :-)